src/sites/twitch-twilight/modules/chat/input.jsx

```diff
@@ -324,10 +324,18 @@ export default class Input extends Module {
 		inst.tempInput = '';
 		inst.messageHistoryPos = -1;
 
+		let element = this.fine.getChildNode(inst);
+		let container = element && element.querySelector('div[class="Layout-sc-nxg1ff-0 chat-input__input-icons"]');
+		container.append(<div class="Layout-sc-nxg1ff-0 appendChildForWrittenChars"><p id="textForCharsWritten">( 0 / 500 )</p></div>);
+		let replacedDivP = element && element.querySelector('p[id="textForCharsWritten"]');
+
 		inst.onKeyDown = function(event) {
 			try {
 				const code = event.charCode || event.keyCode;
 
+				let charsWritten = '( ' + inst.chatInputRef.value.length.toString() + '/500 )';
+				replacedDivP.innerHTML = charsWritten;
```

Comment:
Always use `textContent` rather than `innerHTML` when at all possible to avoid possible security issues. While this specific example couldn't cause issues, innerHTML is a bad habit to get into.